src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,49 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
-);
+export const MovieCard = ({ movie }) => {
+  const { imgUrl, title, description, imdbUrl } = movie;
+
+  return (
+    <div className="card">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            src={imgUrl}
+            alt="Film logo"
+          />
+        </figure>
+      </div>
+
+      <div className="card-content">
+        <div className="media">
+          <div className="media-left">
+            <figure className="image is-48x48">
+              <img src="/images/imdb-logo.jpeg" alt="imdb" />
+            </figure>
+          </div>
+
+          <div className="media-content">
+            <p className="title is-8">{title}</p>
+          </div>
+        </div>
+
+        <div className="content">
+          {description}
+          <br />
+          <a href={imdbUrl}>IMDB</a>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+MovieCard.propTypes = {
+  movie: PropTypes.shape({
+    imgUrl: PropTypes.string,
+    title: PropTypes.string,
+    description: PropTypes.string,
```

Comment:
it's not required, read the task

Reply:
Added default value in dectructur

Reply:
Can I remove isRequired from only description?

Reply:
![image](https://user-images.githubusercontent.com/44294945/74453740-7e068a80-4e8b-11ea-890d-cc07a44cf572.png)